app/controllers/editions_controller.rb

```diff
@@ -71,6 +93,14 @@ def setup_view_paths
     setup_view_paths_for(resource)
   end
 
+  def update_action_is_publish?
+    attempted_activity == :publish
+  end
+
+  def attempted_activity
```

Comment:
I think this method can be made `private` (which it is in the legacy controller).